tools/llcstat.py

```diff
@@ -41,16 +46,19 @@
 
 struct key_t {
     int cpu;
-    int pid;
+    u32 pid;
+    TIDINFO
     char name[TASK_COMM_LEN];
 };
 
 BPF_HASH(ref_count, struct key_t);
 BPF_HASH(miss_count, struct key_t);
 
 static inline __attribute__((always_inline)) void get_key(struct key_t* key) {
+    u64 pid_tgid = bpf_get_current_pid_tgid();
     key->cpu = bpf_get_smp_processor_id();
-    key->pid = bpf_get_current_pid_tgid() >> 32;
+    key->pid = pid_tgid >> 32;
+    GET_TID
```

Comment:
similarly to the other `bpf_text.replace`-related comment, making the replacement as simple as possible will make this easier to understand

```python
if (GET_TID)
  key->tid = (u32)pid_tgid;
```

have the replacement just be `1` or `0`
```

Reply:
DONE